pkgs/by-name/ka/karlyriceditor/package.nix

```diff
@@ -0,0 +1,55 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  qt6,
+  ffmpeg_4,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "karlyriceditor";
+  version = "3.3";
+
+  src = fetchFromGitHub {
+    owner = "gyunaev";
+    repo = finalAttrs.pname;
```

Comment:
Please change this to the literal `"karlyriceditor"` value.
`repo` shouldn't care about what pname's value is.